pages/docs/visualizing-data/calculations.mdx

```diff
@@ -1,33 +1,68 @@
+|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------|
+| **Rolling Average** <br /> Useful for helping smooth out noise in a dataset <br /> <br /> **Rolling Sum** <br /> For looking at the cumulative sum of a measure                                                                                                         | _Periods_ - The number of periods to include in the calculation (i.e. the size of the moving window). <br /> <br /> _Minimum periods_ - The minimum number of observations required for the calculation to be defined for a certain point. For example if you take a seven day moving average, but there are only five days of data before this date, then the calculation will not be defined for that day. Defaults to the same as periods <br /> <br /> _Anchor_ - Whether to set labels at the center of the window or at the edge. Values are `Trailing`, `Centered`, or `Leading`. For a timeseries chart, you will usually want this to be a `Trailing` average or sum. | `rollingAverage(measure, period, anchor, minimumPeriod)`<br /><br />`rollingSum(measure, period, anchor, minimumPeriod)` |
```

Comment:
Not a blocker— we can make this change any time

I suppose this is an existing issue, but the table layout is a bit hard to read:

![CleanShot 2024-08-05 at 10 44 00@2x](https://github.com/user-attachments/assets/9d960133-070d-4df1-ab72-72f6f6d0dfd0)

Maybe it would make sense to write this more like api docs? Not an exact one for one but I like the way radix documents their components: https://www.radix-ui.com/primitives/docs/components/accordion

Something like this might be easier to parse:

### function name
`function example`

Description

table of parameters with columns of name, type, description

Should we also show the available arithmetic operators?

Reply:
I rewrote this like the radix docs – the nice thing is that you can now also use the side bar to navigate to the docs of the calculation you want to see 

<img width="258" alt="Screenshot 2024-08-05 at 11 40 55 AM" src="https://github.com/user-attachments/assets/70000db9-3b79-48cc-b56a-14cdcacae973">